capstone/src/main/resources/static/user.js

```diff
@@ -1,26 +1,23 @@
 /* get the user information for the profile page */
 function getUser(fillForm) {
-  var userType = sessionStorage.getItem("user-type");
-
-  // if there is no userType stored in this session, that means this is a new user
-  if (userType == null) {
-    getUserType(true);
-  } else {
-    if (userType == "organization") {
-      fetch('get-organization').then(response => response.json()).then((data) => {
-        createProfile(data[0], fillForm, true);
+  fetch('user-info').then(response => response.json()).then((data) => {
+    // if there is no data returned, that means this is a new user
+    if (data.userType == "null") {
+      displayMain(false);
+      displayForm("individual", true);
+    } else {
+      if (data.userType == "organization") {
+        createProfile(data, fillForm, true);
         displayNavToggle("individual-nav", "org-nav");
-        sessionStorage.setItem("user-type", data[0].userType);
-      });
-    } else if (userType == "individual") {
-      fetch('get-individual').then(response => response.json()).then((data) => {
-        createProfile(data[0], fillForm, false);
+        sessionStorage.setItem("user-type", data.userType);
```

Comment:
These three lines are the same as in `else` clause; the only differences are the class tyle and true/false parameter in `createProfile`. So the whole if/else block could become one line invocation of a helper method. You don't even need to pass the `individual-nav`/`org-nav` class type -- the helper method can concatenate `userType` with `"-nav"`.